server/middlewares/UserValidation.js

```diff
@@ -212,18 +212,16 @@ class UserValidation {
     * @static
     */
   static validateUserLogin(req, res, next) {
-    const { email, password } = req.body;
+    const { email } = req.body;
     const errors = {};
     const returnError = () => res.status(422).json({
       errors
     });
-    if (email && password) return next();
-    if (!email && !password) {
+    if (email) return next();
```

Comment:
Are we validating that the email is an email and not just a string or existing?

Reply:
Should be doing both. This is fixed.